Makefile

```diff
@@ -95,7 +95,10 @@ crds.clean:
 	@find $(CRD_DIR) -name '*.yaml.sed' -delete || $(FAIL)
 	@$(OK) cleaned generated CRDs
 
-generate.run: gen-kustomize-crds gen-install-doc
+generate.run: gen-kustomize-crds gen-install-doc gen-chart-license
+
+gen-chart-license:
+    @cp LICENSE cluster/charts/crossplane/LICENSE
```

Comment:
Not seem to be relevant to the CI failure but, should add `-r` here? https://www.tecmint.com/force-cp-to-overwrite-without-confirmation/

Reply:
We shouldn't have a problem just using `cp`. (I've tested this locally).
That said I don't think there's any harm in using `-f` flag to force overwrite. 

`-r` would be needed if we were copying directories.

Reply:
Actually, that would only be a precaution to the following statement in the linked blog above: 

> By default, modern Linux distributions especially those in the Red Hat Enterprise Linux (RHEL) family come with an alias for the cp command which makes a user run the cp command in interactive mode. This may not be the case on Debian and Ubuntu derivatives.

Reply:
Ah, sorry, I meant `-f`, yes 🤦